src/Components/MessageForm.jsx

```diff
@@ -0,0 +1,87 @@
+import { useState, useEffect } from "react";
+
+export const MessageForm = ({ addNewMessage, fetchMessages }) => {
+    /* state variables */
+    const [newMessage, setNewMessage] = useState("");
+    const [errorMessage, setErrorMessage] = useState("");
+
+    // Initialize a variable to store the API for thoughts
+    const thoughtAPI = "https://happy-thoughts-ux7hkzgmwa-uc.a.run.app/thoughts";
+
+    /* Function to handle change in text area */
+    const handleTextAreaChange = (e) => {
+        setNewMessage(e.target.value);
+    }
+
+    /* Function to check if the message is too long */
+    useEffect(() => {
+        if (newMessage.length >= 141) {
```

Comment:
since I can't see, did you have a countdown under the insert textarea?

Reply:
I do, it works as per the instructions